### ISSUE TITLE
BLE scan loop silently fails on Bleak errors/exceptions

### ISSUE THREAD

**Issue by JiddoDev** [theengs/gateway#178]
**Describe the bug**
If the BleakScanner runs raises an exception during scanning the entire BLE scan loop will silently fail, leaving the user only with the "Starting BLE scan" message that indicates normal operations. 

Turns out that this error was being triggered for me:
`[org.freedesktop.DBus.Error.AccessDenied] An AppArmor policy prevents this sender from sending this message to this recipient; type="method_call", sender="(null)" (inactive) interface="org.freedesktop.DBus" member="AddMatch" error name="(unset)" requested_reply="0" destination="org.freedesktop.DBus" (bus)`

Caused by not running my gateway Docker container as privileged. There was no indication of this error until I added exception debug printing to the gateway files that ran inside the container. 

**To Reproduce**
1. Run TheengsGateway as a Docker container that is not privileged and does not have a custom AppArmor policy.
2. Note the log ouput is limited to the following output and no messages get sent to MQTT:
```
INFO:BLEGateway:Starting BLE scan
INFO:BLEGateway:Connected to MQTT Broker!
INFO:BLEGateway:Subscribed to home/TheengsGateway/commands
```

**Expected behavior**
I expected log output to clearly indicate if BLE scanning fails or is interrupted in some way that will prevent the gateway from working.

**Environment**
[Theengs-diagnostics.txt](https://github.com/theengs/gateway/files/13402908/Theengs-diagnostics.txt)

**Additional context**
[Theengs-docker-compose.txt](https://github.com/theengs/gateway/files/13402911/Theengs-docker-compose.txt)

**Comment by koenvervloesem** [comment]
Good point, the current behavior is not user-friendly. Does PR https://github.com/theengs/gateway/pull/179 work better in your situation?

**Comment by JiddoDev** [comment]
Sorry for the delay. Finally had the time to verify the fix in #179. It works as I'd expect. Running without privileged mode now gives me the following output:

```
INFO:BLEGateway:Starting BLE scan
ERROR:BLEGateway:[org.freedesktop.DBus.Error.AccessDenied] An AppArmor policy prevents this sender from sending this message to this recipient; type="method_call", sender="(null)" (inactive) interface="org.freedesktop.DBus" member="AddMatch" error name="(unset)" requested_reply="0" destination="org.freedesktop.DBus" (bus)
Traceback (most recent call last):
  File "/opt/venv/lib/python3.9/site-packages/TheengsGateway/ble_gateway.py", line 374, in ble_scan_loop
    await scanner.start()
  File "/opt/venv/lib/python3.9/site-packages/bleak/__init__.py", line 198, in start
    await self._backend.start()
  File "/opt/venv/lib/python3.9/site-packages/bleak/backends/bluezdbus/scanner.py", line 168, in start
    manager = await get_global_bluez_manager()
  File "/opt/venv/lib/python3.9/site-packages/bleak/backends/bluezdbus/manager.py", line 1046, in get_global_bluez_manager
    await instance.async_init()
  File "/opt/venv/lib/python3.9/site-packages/bleak/backends/bluezdbus/manager.py", line 249, in async_init
    assert_reply(reply)
  File "/opt/venv/lib/python3.9/site-packages/bleak/backends/bluezdbus/utils.py", line 22, in assert_reply
    raise BleakDBusError(reply.error_name, reply.body)
bleak.exc.BleakDBusError: [org.freedesktop.DBus.Error.AccessDenied] An AppArmor policy prevents this sender from sending this message to this recipient; type="method_call", sender="(null)" (inactive) interface="org.freedesktop.DBus" member="AddMatch" error name="(unset)" requested_reply="0" destination="org.freedesktop.DBus" (bus)
INFO:BLEGateway:Sent 0 messages to MQTT
ERROR:BLEGateway:BLE scan loop stopped
```
This is a big improvement over silent failure. Thanks!

**Comment by koenvervloesem** [comment]
Good to hear, the fix is now also in the recent 1.2 release.
